src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -60,7 +60,11 @@ public LocalEpisode GetLocalEpisode(string filename, Series series, ParsedEpisod
                 parsedEpisodeInfo = Parser.ParsePath(filename);
             }
 
-            if (parsedEpisodeInfo == null || parsedEpisodeInfo.IsPossibleSpecialEpisode)
+            if (parsedEpisodeInfo != null && parsedEpisodeInfo.IsPossibleSceneSeasonSpecial && GetStandardEpisodes(series, parsedEpisodeInfo, true, null).Any())
+            {
+                // SxxE00 episodes are sometimes mapped via TheXEM, don't use episode title parsing in that case.
```

Comment:
What's the purpose of this `if`? It's not preventing falling into an `else` and it doesn't do anything on it's own.

Reply:
It prevents falling into the elseif, IF the E00 episode is mapped by TheXEM. This causes it to bypass the FindByTitle and go straight to the standard handling.

But if that's unclear based on reading the code, then it needs to be refactored. I'll see what I can do to clear it up.